Exiled.Events/Patches/Events/Player/SearchPickupRequesting.cs

```diff
@@ -0,0 +1,76 @@
+// -----------------------------------------------------------------------
+// <copyright file="SearchPickupRequesting.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Player
+{
+#pragma warning disable SA1313
+    using System;
+
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using InventorySystem.Searching;
+
+    using Mirror;
+    using Mirror.LiteNetLib4Mirror;
+
+    /// <summary>
+    /// Patches <see cref="SearchCoordinator.ReceiveRequestUnsafe"/>.
+    /// Adds the <see cref="Handlers.Player.SearchPickupRequesting"/> event.
+    /// </summary>
+    [HarmonyPatch(typeof(SearchCoordinator), nameof(SearchCoordinator.ReceiveRequestUnsafe))]
+    internal static class SearchPickupRequesting
+    {
+        private static void Prefix(SearchCoordinator __instance, ref bool __result, out SearchSession? session, out SearchCompletor completor)
```

Comment:
Return bool value here

Reply:
it's done i return the bool value for make the base method not call when the Prefix have any error